src/org/usfirst/frc/team1923/robot/commands/drive/GyroTurnCommand.java

```diff
@@ -62,13 +65,16 @@ protected void initialize() {
 
     @Override
     protected void execute() {
-        // Debug
+
         if (RobotMap.DEBUG) {
             double currentAngle = this.target.pidGet();
             System.out.println("target = " + this.degrees + ", current = " + currentAngle + ", error = " + (this.degrees - currentAngle));
         }
     }
 
+    /**
+     * finished when - timed out or - reached target heading

```

Comment:
Same as above, please remove these types of comments as it is below the assumed competence of the audience. (Will not mark any others, but please fix them)